blob/fileblob/fileblob_test.go

```diff
@@ -36,7 +36,7 @@ func makeBucket(t *testing.T) (*blob.Bucket, func()) {
 	if err != nil {
 		t.Fatal(err)
 	}
-	return b, func() {}
+	return b, func() { _ = os.RemoveAll(dir) }
```

Comment:
The body can just be `os.RemoveAll(dir)`

Reply:
The linter that Travis uses complains about that, unfortunately, something like "Error value not being checked."